from datalab_sdk.api.exceptions import InputNotValid, ResourceNotFound

class SirNotFound(ResourceNotFound):
    """Raised when the requested info not found in sir"""
    def __init__(self, correlation_id: str = None):
        super().__init__(f"The requested info for sir not found", correlation_id)